app/User.php

```diff
@@ -14,7 +14,7 @@ class User extends Authenticatable
     /**
      * The attributes that are mass assignable.
      *
-     * @var array<string>
+     * @var array<int, string>
```

Comment:
nit: `list<string>` or `string[]` is better. `array<int, string>` implies the int has other semantics. We are stuck with `array<int, string>` since that base class (Laravel) uses it.